src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,47 @@
+package duke;
+
+/**
+ * Class to take care of UI
+ */
+public class Ui {
+    private String divider = "------------------------------------\n";

```

Comment:
could the following fields be FINAL, as they are pretty much constants that are not changes in the code